### DISCUSSION TITLE
Suggest how to grow up dota community again

### DISCUSSION THREAD

**Discussion by dragonuser1** [ValveSoftware/Dota2-Gameplay#14509] [Ideas]
Can Valve do new things in front of other companies?

The community and gamedesign team needs to innovate in some way, please consider using the same compedium idea in the global server ladders.

But instead of using the compedium, they could use Dotaplus, where part of the resources would be offered to well-ranked people.

Not focusing on pro-players but on a stable prize pool that perhaps reaches the top 300.

Ladders in menu game, promoting the player and other campaigns that aim to increase the number of players.

**Comment by nyczalex** [comment]
Yes, I agree, they really need to grow the dota community again. While for sure, priority should always be on the game updates itself and keeping it fresh and competitive for what the game is known for. 

Some suggestions are:
- Global compendium annually.
- Guild updates for leaders/moderators to follow up on players contributions and improvements as a player, sorta easier way to track player stats. This will allow guilds to have their own little inhouse guild tournaments and what not for a more fun&competitive atmospher/level.
- Fix on chests gamble rate. The % of it is so low that it is 100% cheaper to obtain on market place for 99%+ of players manually opening. Make it to a % close enough for people to actually want to gamble and make it profit from it like other games, rather than now, being always cheaper to pick up directly from the market. 
- More incentives for people to grind. This goes hand to hand with improving the gamba rate on chest openings. With a higher rate of return on chest openings, the valve of chests will stabilize more and go up. People are able to grind and farm chests and sell them on the market for others that cant and/or want to gamble. 
- Update to Dota Plus. Play it out like a mini compendium and steam point shop. Give it more bang for its buck. More quests, more ways to earn, more ways to spend.
- Shifting away from the Dota theme.. Over the years, we already started with models such as chibi characters, color arcanas, etc.. I remember back then when all these effects i've mentioned, there was a lot of negative feedback but now fastforward to current, we have it all, chibi cuteness, color arcanas, interactive headshot bullseye sniper, list goes on..  Keep up the good work and bring out the full bang and get back to marketing with these cute skins.

At the end, the game is still brilliant all these years and that should always be the priority but let's be real, in this age of gaming, cosmetics is part of a game and people enjoy. While there are people that say they don't care, i've had never ever seen a default hero skin in a ranked match.
,
